specification/servicenetworking/ServiceNetworking.Management/main.tsp

```diff
@@ -87,6 +87,58 @@ model AssociationSubnet {
   id: string;
 }
 
+//----------------------- SecurityPolicy -----------------------
+@doc("SecurityPolicy Subresource of Traffic Controller.")
+@parentResource(TrafficController)
+model SecurityPolicy is TrackedResource<SecurityPolicyProperties> {
+  @key("SecurityPolicy")
+  @doc("SecurityPolicy")
+  @visibility("read")
+  @path
+  @segment("securityPolicy")
+  @pattern("^[A-Za-z0-9]([A-Za-z0-9-_.]{0,62}[A-Za-z0-9])?$")
+  name: string;
+}
+
+@doc("SecurityPolicy Properties.")
+model SecurityPolicyProperties {
+  @doc("policy Type")
+  policyType: PolicyType;
+
+
+  wafPolicy?: WAFPolicy;
```

Comment:
I'd expect that you should put `@doc` descriptions for all your properties

Reply:
sure, I've added docs for all the properties

Reply:
I don't see a new iteration. Did you forget to push changes?